src/eventloop.jl

```diff
@@ -52,7 +57,63 @@ function waitloop()
     end
 end
 
-const idle_delay = Ref(0.5)
-function set_idle_delay(delay_secs::Float64=0.5)
-    idle_delay[] = delay_secs
+#=
+    Hold Channel Interface
+    This allows user processes to register and deregister
+    channels that allow user code to hold IJulia from sending
+    the idle status message until signaled.  This would have
+    been prettier if there were level-triggered events in
+    Julia base, but this will do okay for now.
```

Comment:
A lot of the time when a language/library supports a Condition type, right next to it in the docs will be an Event type that is just for level triggering.  Unbuffered channels work for it, but they aren't really made for that purpose.